File: Latihan 3/python/Processor.py
class Processor():

    __name = ""
    __price = ""

    def __init__(self, name = "", price = ""):
        self.__name = name
        self.__price = price

    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price

    def printProcessor(self):
        print("Processor : ", str(self.getName()))
        print("Country : ", str(self.getPrice()))

File: Latihan 3/c++/processor.cpp
#include <iostream>
#include <string>

using namespace std;

class processor{
    private:
    string name;
    int price;

    public:
    processor(){
        this->name = "";
        this->price = 0;
    }

    processor(string name, int price){
        this->name = name;
        this->price = price;
    }

    void setName(string name){
        name = name;
    }

    void setprice(int price){
        this->price = price;
    }

    string getName(){
        return this->name;
    }

    int getprice(){
        return this->price;
    }

    void printProcessor(){
        cout << "Processor       : " << this->name << endl;
        cout << "Processor price : " << this->price << endl;
    }

    ~processor(){}
};